samples/BlobSamples.php

```diff
@@ -413,12 +424,19 @@ function listBlobsSample($blobClient)
 {
     try {
         // List blobs.
-        $blob_list = $blobClient->listBlobs("mycontainer");
-        $blobs = $blob_list->getBlobs();
-    
-        foreach ($blobs as $blob) {
-            echo $blob->getName().": ".$blob->getUrl().PHP_EOL;
-        }
+        $listBlobsOptions = new ListBlobsOptions();
+        $listBlobsOptions->setPrefix("myblob");
+        $listBlobsOptions->setMaxResults(1);
```

Comment:
Better to have a comment to clarify that setting max result to 1 is just to demonstrate the continuation token. It is not the recommended value in product.

Reply:
done